patefon.co,"HTTPSConnectionPool(host='patefon.co', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='patefon.co', port=443): Read timed out. (read timeout=2)"
patmar.ee,"HTTPSConnectionPool(host='patmar.ee', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='patmar.ee', port=443): Read timed out. (read timeout=2)"
patricinhaesperta.com.br,"HTTPSConnectionPool(host='patricinhaesperta.com.br', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='patricinhaesperta.com.br', port=443): Read timed out. (read timeout=2)"
patro.cz,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
paysera.com,"HTTPSConnectionPool(host='www.paysera.pl', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.paysera.pl', port=443): Read timed out. (read timeout=2)"
pcfarm.ro,"HTTPSConnectionPool(host='pcfarm.ro', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: UNSUPPORTED_PROTOCOL] unsupported protocol (_ssl.c:1124)')))","HTTPSConnectionPool(host='pcfarm.ro', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: UNSUPPORTED_PROTOCOL] unsupported protocol (_ssl.c:1124)')))"
pctown.co.nz,"HTTPSConnectionPool(host='pctown.co.nz', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='pctown.co.nz', port=80): Read timed out. (read timeout=2)"
pdn.ac.lk,"HTTPSConnectionPool(host='pdn.ac.lk', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f681cdae0a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='pdn.ac.lk', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f681cdb7790>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
peatus.ee,"HTTPSConnectionPool(host='web.peatus.ee', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='web.peatus.ee', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
pedagogica.edu.co,"HTTPSConnectionPool(host='pedagogica.edu.co', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f681cea9490>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='pedagogica.edu.co', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f681d04bcd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
pegem.net,"HTTPSConnectionPool(host='www.pegem.net', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.pegem.net', port=443): Read timed out. (read timeout=2)"
peika.bg,"HTTPSConnectionPool(host='www.peika.bg', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.peika.bg', port=443): Read timed out. (read timeout=2)"
pellervo.fi,"HTTPSConnectionPool(host='pellervo.fi', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='pellervo.fi', port=443): Read timed out. (read timeout=2)"
penang.gov.my,"HTTPSConnectionPool(host='penang.gov.my', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f681cdbd550>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='penang.gov.my', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f681cdbd760>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
peoplesbank.lk,"HTTPSConnectionPool(host='peoplesbank.lk', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f681d05c100>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='peoplesbank.lk', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f681d05c610>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
perach.org.il,"HTTPSConnectionPool(host='perach.org.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='perach.org.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
perak.gov.my,"HTTPSConnectionPool(host='perak.gov.my', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f681cd60b20>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='perak.gov.my', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f681cd30d30>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
perfil.com,"HTTPSConnectionPool(host='www.perfil.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.perfil.com', port=443): Read timed out. (read timeout=2)"
periodicobolivia.com.bo,"HTTPSConnectionPool(host='periodicobolivia.com.bo', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f681d074580>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='periodicobolivia.com.bo', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f681cc13d00>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
peritiuniti.it,Exceeded 30 redirects.,Exceeded 30 redirects.
persee.fr,"HTTPSConnectionPool(host='www.persee.fr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.persee.fr', port=443): Read timed out. (read timeout=2)"
pertanian.go.id,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
pgdlisboa.pt,"HTTPSConnectionPool(host='pgdlisboa.pt', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f681c58df40>, 'Connection to pgdlisboa.pt timed out. (connect timeout=2)'))","HTTPConnectionPool(host='pgdlisboa.pt', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f681c40b0a0>, 'Connection to pgdlisboa.pt timed out. (connect timeout=2)'))"
pgr.gob.ni,"HTTPSConnectionPool(host='pgr.gob.ni', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f681c538370>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='pgr.gob.ni', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f681c47f490>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
pharmaline.co.il,"HTTPSConnectionPool(host='www.pharmaline.co.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.pharmaline.co.il', port=443): Read timed out. (read timeout=2)"
pharmilion.gr,"HTTPSConnectionPool(host='pharmilion.gr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='pharmilion.gr', port=443): Read timed out. (read timeout=2)"
phimmoizz.net,"HTTPSConnectionPool(host='phimmoizz.net', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='phimmoizz.net', port=80): Read timed out. (read timeout=2)"
pichinchacomunicaciones.com.ec,"HTTPSConnectionPool(host='pichinchacomunicaciones.com.ec', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f681ca019a0>, 'Connection to pichinchacomunicaciones.com.ec timed out. (connect timeout=2)'))","HTTPConnectionPool(host='pichinchacomunicaciones.com.ec', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f681c4fab50>, 'Connection to pichinchacomunicaciones.com.ec timed out. (connect timeout=2)'))"
pietiek.com,"HTTPSConnectionPool(host='pietiek.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='pietiek.com', port=443): Read timed out. (read timeout=2)"
pikabu.ru,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
pikiwiki.org.il,"HTTPSConnectionPool(host='www.pikiwiki.org.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.pikiwiki.org.il', port=443): Read timed out. (read timeout=2)"
pikk.ee,"HTTPSConnectionPool(host='www.pikk.ee', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.pikk.ee', port=443): Read timed out. (read timeout=2)"
pirkka.fi,"HTTPSConnectionPool(host='pirkka.fi', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.pirkka.fi', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
pitat.com,"HTTPSConnectionPool(host='pitat.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f681bdc5760>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='pitat.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f681c569040>: Failed to establish a new connection: [Errno 111] Connection refused'))"
pizzasnella.it,"HTTPSConnectionPool(host='pizzasnella.it', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f681c966250>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='www.pizzasnella.it', port=80): Read timed out. (read timeout=2)"
planalto.gov.br,"HTTPSConnectionPool(host='planalto.gov.br', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1124)')))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
planetenjorden.dk,"HTTPSConnectionPool(host='planetenjorden.dk', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f681ce86790>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='planetenjorden.dk', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f681bd1d9d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
pmda.go.jp,"HTTPSConnectionPool(host='pmda.go.jp', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f681c47fb50>, 'Connection to pmda.go.jp timed out. (connect timeout=2)'))","HTTPConnectionPool(host='pmda.go.jp', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f681ceea2b0>, 'Connection to pmda.go.jp timed out. (connect timeout=2)'))"
pnc.gob.sv,"HTTPSConnectionPool(host='pnc.gob.sv', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f681bd628b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='pnc.gob.sv', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f681c8c9a00>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
pnp.co.za,"HTTPSConnectionPool(host='pnp.co.za', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f681c58d4f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='pnp.co.za', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f681bd9d4f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
poder-judicial.go.cr,"HTTPSConnectionPool(host='poder-judicial.go.cr', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f681c834160>, 'Connection to poder-judicial.go.cr timed out. (connect timeout=2)'))","HTTPConnectionPool(host='poder-judicial.go.cr', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f681c259820>, 'Connection to poder-judicial.go.cr timed out. (connect timeout=2)'))"
poderjudicial.gob.ni,"HTTPSConnectionPool(host='poderjudicial.gob.ni', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f681cd4d640>, 'Connection to poderjudicial.gob.ni timed out. (connect timeout=2)'))","HTTPConnectionPool(host='poderjudicial.gob.ni', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f681bd24ee0>, 'Connection to poderjudicial.gob.ni timed out. (connect timeout=2)'))"
poduckun.net,"HTTPSConnectionPool(host='poduckun.net', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='poduckun.net', port=443): Read timed out. (read timeout=2)"
pofta-buna.com,"HTTPSConnectionPool(host='pofta-buna.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f681d6a07f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='pofta-buna.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f681d6ab6a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
poipeplekkers.be,"HTTPSConnectionPool(host='poipeplekkers.be', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f681c49cb50>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='poipeplekkers.be', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f681cd6be50>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
poisci.me,"HTTPSConnectionPool(host='poisci.me', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='poisci.me', port=80): Read timed out. (read timeout=2)"
polazak.rs,Exceeded 30 redirects.,Exceeded 30 redirects.
policia.gov.co,"HTTPSConnectionPool(host='policia.gov.co', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f681cbbf3d0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='policia.gov.co', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f681cd18fd0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
politikaplus.com,"HTTPSConnectionPool(host='politikaplus.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='politikaplus.com', port=80): Read timed out. (read timeout=2)"
polito.it,"HTTPSConnectionPool(host='polito.it', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f681cd12220>, 'Connection to polito.it timed out. (connect timeout=2)'))","HTTPConnectionPool(host='polito.it', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f681c966580>, 'Connection to polito.it timed out. (connect timeout=2)'))"
polsl.pl,"HTTPSConnectionPool(host='polsl.pl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f681cd30400>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='polsl.pl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f681cd07610>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
pom.go.id,"HTTPSConnectionPool(host='pom.go.id', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='pom.go.id', port=443): Max retries exceeded with url: /new/ (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
pontofrio.com.br,"HTTPSConnectionPool(host='www.pontofrio.com.br', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.pontofrio.com.br', port=443): Read timed out. (read timeout=2)"
popasia.net,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
popcornfor2.com,"HTTPSConnectionPool(host='popcornfor2.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='popcornfor2.com', port=80): Read timed out. (read timeout=2)"
porezna-uprava.hr,"HTTPSConnectionPool(host='porezna-uprava.hr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f681c548b50>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='porezna-uprava.hr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f681c580df0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
portal.gov.bd,"HTTPSConnectionPool(host='portal.gov.bd', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='portal.gov.bd', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
portaldasfinancas.gov.pt,"HTTPSConnectionPool(host='portaldasfinancas.gov.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f681cd4dfd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='portaldasfinancas.gov.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f681c83fac0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
portorecanaticalcio.it,"HTTPSConnectionPool(host='www.portorecanaticalcio.it', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.portorecanaticalcio.it', port=443): Read timed out. (read timeout=2)"
portugal.gov.pt,"HTTPSConnectionPool(host='www.portugal.gov.pt', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.portugal.gov.pt', port=443): Read timed out. (read timeout=2)"
posta.com.tr,"HTTPSConnectionPool(host='posta.com.tr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.posta.com.tr', port=443): Read timed out. (read timeout=2)"
posted.co.rs,"HTTPSConnectionPool(host='posted.co.rs', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f681c46d6a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='posted.co.rs', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f681cd60f40>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
pozitivke.net,"HTTPSConnectionPool(host='pozitivke.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'pozitivke.net' doesn't match either of '*.mojhost.net', 'mojhost.net'"")))","HTTPConnectionPool(host='pozitivke.net', port=80): Read timed out. (read timeout=2)"
pozri.sk,"HTTPSConnectionPool(host='pozri.sk', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='pozri.sk', port=80): Read timed out. (read timeout=2)"
